File: app/src/main/java/com/blockeq/stellarwallet/interfaces/AfterTextChanged.java
package com.blockeq.stellarwallet.interfaces;

import android.text.Editable;
import android.text.TextWatcher;

public abstract class AfterTextChanged implements TextWatcher {
    @Override
    public void beforeTextChanged(CharSequence s, int start, int count, int after) {
        // empty implementation
    }

    @Override
    public void onTextChanged(CharSequence s, int start, int before, int count) {
        // empty implementation
    }

    @Override
    public abstract void afterTextChanged(Editable s);
}

File: app/src/main/java/com/blockeq/stellarwallet/interfaces/AfterTextChanged.kt
package com.blockeq.stellarwallet.interfaces

import android.text.Editable
import android.text.TextWatcher

abstract class AfterTextChanged : TextWatcher {
    override fun beforeTextChanged(s: CharSequence, start: Int, count: Int, after: Int) {
        // empty implementation
    }

    override fun onTextChanged(s: CharSequence, start: Int, before: Int, count: Int) {
        // empty implementation
    }

    abstract override fun afterTextChanged(editable: Editable)
}
